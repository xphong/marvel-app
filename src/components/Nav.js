import React from 'react';
import { IndexLink, Link } from 'react-router';

const Nav = () => (
  <div className='ui fixed menu'>
    <div className='ui container'>
      <div className='header item'>
        Marvel Search App
      </div>
      <IndexLink to='/' className='item' activeClassName='route--active'>
        Home
      </IndexLink>
      <Link to='/powerlevels' className='item' activeClassName='route--active'>
        Power Levels
      </Link>
      <Link to='/about' className='item' activeClassName='route--active'>
        About
      </Link>
    </div>
  </div>
);

export default Nav;
