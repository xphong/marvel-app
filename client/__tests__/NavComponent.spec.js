import React from 'react';

import Nav from '../components/ui/Nav';

describe('Nav Component', () => {
  it('should render Nav', () => {
    const wrapper = shallow(
      <Nav />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
