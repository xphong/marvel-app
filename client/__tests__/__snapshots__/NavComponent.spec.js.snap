// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component should render Nav 1`] = `
<div
  className="ui fixed menu"
>
  <div
    className="ui container"
  >
    <div
      className="header item"
    >
      Marvel App
    </div>
    <IndexLink
      activeClassName="route--active"
      className="item"
      to="/"
    >
      Home
    </IndexLink>
    <Link
      activeClassName="route--active"
      className="item"
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/powerlevels"
    >
      Power Levels
    </Link>
    <Link
      activeClassName="route--active"
      className="item"
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/about"
    >
      About
    </Link>
  </div>
</div>
`;
